app/helpers/view_components/student/track_nav.rb

```diff
@@ -0,0 +1,61 @@
+module ViewComponents
```

Comment:
What is the advantage of using something like this compared to a view partial?

Reply:
So:
- I think there is value in having a component that determines logical parts of things (for example here it works out the tab classes, but will also add more details to the tabs (e.g. "concepts" will change to "10/100 concepts"). It also means that can be easily tested.
- However, the `to_s` method itself could be rendering a partial instead, with data, or locals, passed into it. A little like we do for Kaido Reports. I actually started by doing that then rolled back to this, just for the sake of simplicity. But I imagine as I do a few more they can turn into partials.